azure-pipelines.yml

```diff
@@ -591,7 +591,7 @@ jobs:
       set -e
       . util/build_consts.sh
       module load "xilinx/vivado/$(VIVADO_VERSION)"
-      ci/scripts/run-fpga-tests.sh hyper310 cw310_sival,-broken || { res=$?; echo "To reproduce failures locally, follow the instructions at https://opentitan.org/book/doc/getting_started/setup_fpga.html#reproducing-fpga-ci-failures-locally"; exit "${res}"; }
```

Comment:
Are you sure we don't want to keep `-broken`?

Reply:
run-fpga-tests.sh will add `,-broken,-skip_in_ci` so we don't need to explicit specify it.